test/src/pelz_test.c

```diff
@@ -28,11 +31,33 @@ int clean_suite(void)
 //Main function for the unit testing of the Pelz Service application
 int main(int argc, char **argv)
 {
+  char cwd[1024];
+  char *tmp_id;
+  CharBuf id;
+  char *key[6] = { "KIENJCDNHVIJERLMALIDFEKIUFDALJFG", "KALIENGVBIZSAIXKDNRUEHFMDDUHVKAN", "HVIJERLMALIDFKDN",
+		   "NGVBIZSAIXKDNRUE", "EKIUFDALVBIZSAIXKDNRUEHV", "ALIENGVBCDNHVIJESAIXEKIU"};
+
   set_app_name("pelz");
   set_app_version("0.0.0");
   set_applog_path("./test/log/pelz.log");
   set_applog_severity_threshold(LOG_INFO);
 
+  getcwd(cwd, sizeof(cwd));
```

Comment:
Do you even need to get the current working directory? Won't `fopen` work just fine with a relative pathname from the current directory?

Reply:
As I look further, it might be even better to just specify an array of filenames so you don't have to repeat the name creation code later.